### DISCUSSION TITLE
blog/linksys-velop-and-bypass-gateway/

### DISCUSSION THREAD

**Discussion by giscus** [sqybi/SQYBI.com#8] [Announcements]
# blog/linksys-velop-and-bypass-gateway/

TL;DR: 在一个有旁路由的网络环境中使用 Linksys VELOP 路由器作为主路由，可能会导致旁路由搭建困难或损失网络性能，请优先考虑其它路由器。

https://sqybi.com/blog/linksys-velop-and-bypass-gateway/

**Comment by xuejc1988** [comment]
非常同意你的最后一句——“不要折腾网络，会变得不幸。”
我家最稳定的路由器，是早年给我爸妈买的Buffalo，五年间出了ISP出问题找过我两次，平时从不烦我。
第二稳定的是一台在嘉兴房子中无人看管的TP-Link，日常连接冰箱和监控摄像头，完全无人值守。
最不稳定的是我上海的路由器——但这个问题在我搬家后只用ISP赠送的光猫和路由并压抑住自己人性的恶而不去改桥接甚至都没理会散热和信号问题把他们从弱点箱里拿出来后——得到了基本解决。

**Reply by sqybi** [reply]
现在的光猫路由竟然都这么强了吗！

**Comment by xuejc1988** [comment]
看完了，还有一些些没懂，但我觉得楠老师应该在你折腾的时候想把你扔出去……

**Comment by TayYim** [comment]
很好的文章，让我半路放弃继续折腾。特地登录点个赞

**Reply by sqybi** [reply]
哈哈哈哈哈哈哈哈哈！
所以最后还是用了小米的路由器，就让这次折腾作为人生中的一点涟漪随风飘散吧（
